frontend/src/common/constants/user-instance.constant.ts

```diff
@@ -0,0 +1,19 @@
+import { EditUserPayloadKey, UserSex, UserType } from '../enums'
```

Comment:
Not sure that you need in the root. It's local think, so move it near the component where you use it